cmake/modules/FindSUNDIALS.cmake

```diff
@@ -40,7 +52,19 @@ DEAL_II_FIND_LIBRARY(SUNDIALS_LIB_SER NAMES sundials_nvecserial
   PATH_SUFFIXES lib${LIB_SUFFIX} lib64 lib
   )
 
-SET(SUN_INC "${SUNDIALS_DIR}/include")
```

Comment:
```
SET_IF_EMPTY(SUNDIALS_INCLUDE_HINT "${SUNDIALS_DIR}/include")
```
?

then you would not need those other lines

Reply:
ahh, nah, that's not what you need...

Reply:
I doubt that it really matters whether or not we check `/include/` but I figured we should just be conservative and skip that directory if `SUNDIALS_DIR` is not defined.

Reply:
i agree.